plugin/trino-memsql/src/test/java/io/trino/plugin/memsql/TestMemSqlConnectorTest.java

```diff
@@ -89,6 +89,9 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
             case SUPPORTS_ARRAY:
                 return false;
 
+            case SUPPORTS_RENAME_TABLE_ACROSS_SCHEMAS:
```

Comment:
This should also be added to `TestMemSqlLatestConnectorSmokeTest`, sorry I missed that in my commit in #7333